src/Cli/dotnet/commands/dotnet-workload/InstallStateContents.cs

```diff
@@ -0,0 +1,24 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.DotNet.Workloads.Workload
+{
+    internal class InstallStateContents
+    {
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public bool? useWorkloadSets { get; set; }
```

Comment:
You can set up the Json serialization options to use camelCase, and then you can leave the C# properties PascalCase: https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/customize-properties?pivots=dotnet-7-0#use-a-built-in-naming-policy